internal/model/recommendation.go

```diff
@@ -0,0 +1,18 @@
+package model
+
+import (
+	"github.com/lib/pq"
+	"gorm.io/datatypes"
+)
+
+type Recommendation struct {
+	ID              uint   `gorm:"primaryKey;not null;autoIncrement"`
+	ClusterID       string `gorm:"type:text"`
+	Cluster         Cluster
+	ExperimentName  string         `gorm:"type:text"`
+	Namespace       string         `gorm:"type:text"`
+	K8sObjectType   string         `gorm:"type:text"`
+	K8sObjectName   string         `gorm:"type:text"`
```

Comment:
Yes

Reply:
As discussed, let's keep this as consistent as possible with our query params
```
K8sObjectType -> WorkloadType
K8sObjectName -> Workload
```
Additionally, I think K8sObjectType might as well be a string enum type.

Reply:
Gorm auto-migration does not supports adding enum types into postgres. We need to do it via running raw DB query.
For now I have added enum at the code level and it will take care that ros-ocp will not insert any other value.